typing/typemod.ml

```diff
@@ -3443,6 +3447,10 @@ let report_error ~loc _env = function
       Location.errorf ~loc
         "This is an alias for module %a, which is missing"
         (Style.as_inline_code path) p
+  | Cannot_alias p ->
+      Location.errorf ~loc
+        "Cannot alias the module %a, it is a functor argument"
```

Comment:
I would propose to merge the two clauses into `The functor argument %a cannot be aliased`.

Reply:
`Functor arguments like %a cannot be aliased`?